[25기 강단 ] 로그인 페이지 구현 완료,  메인페이지 댓글 기능 구현 중

kangdanne opened the pull request:
[Mission1] 로그인 페이지 레이아웃 (완료)
[Mission2] id, pw 입력 시 로그인 버튼 활성화 기능 (완료)
[Mission3] 메인 페이지 레이아웃 (완료)
[Mission4] 댓글 내용 입력 후 Enter 키, 혹은 게시 버튼 클릭 시 댓글 추가 기능 (완료)

## 수정 사항 
-  main.js 댓글 내용 한글로 입력 시 enter키 keyup때 두 줄로 입력되는 현상 수정
		 
## 수정 사항들 자세한 내용
-  main.js 댓글 내용 한글로 입력 시 enter키 keyup때 두 줄로 입력되는 현상 수정
   - keyup 이벤트를 keypress로 수정

## 기타 질문 및 특이 사항


## 체크 리스트 (아래 사항들이 전부 체크되어야만 merge가 됩니다!)
- [ ] 필요한 test들을 완료하였고 기능이 제대로 실행되는지 확인 하였습니다.
- [x] Wecode의 코드 스타일 가이드에 맞추어 코드를 작성 하였습니다.
- [x] 제가 의도한 파일들과 수정 사항들만 커밋이 된 것을 확인 하였습니다.
- [ ] 본 수정 사항들을 팀원들과 사전에 상의하였고 팀원들 모두 해당 PR에 대하여 알고 있습니다.

joonsikyang commented on students/12th/hongyechan/index.html:
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>repl.it</title>
+    <link href="style.css" rel="stylesheet" type="text/css" />
+  </head>
+  <body>
+    <script src="script.js"></script>
+  </body>
+</html>
```
index.html은 삭제 후 다시 push 해주세요!

hayyim0626 replied:
삭제 후 다시 push 했습니다!

joonsikyang commented on students/12th/hongyechan/login.html:
```diff
@@ -0,0 +1,81 @@
+    
+    <script src="js/login.js"></script>
+  </body>
+</html>
```
세션 시간 때 말씀드린 내용 반영해서 리팩토링 진행해주세요!

hayyim0626 replied:
refactoring 진헹 헸습니다!

joonsikyang commented on students/12th/hongyechan/style/login.css:
```diff
@@ -0,0 +1,95 @@
+* {
+  box-sizing: border-box;
+}
```
이러한 속성은 common.css에서 작성해주시고 html 파일에서 import 해서 사용해주세요.

hayyim0626 replied:
common.css로 이동했습니다!

joonsikyang commented on students/12th/hongyechan/style/login.css:
```diff
@@ -0,0 +1,95 @@
+* {
+  box-sizing: border-box;
+}
+.entire {
+  background-color: rgba(var(--b3f, 250, 250, 250), 1);
+}
```
요소와 요소 사이는 한 줄 띄워주세요 :)

joonsikyang reviewed (commented):
예찬님! 리팩토링 하면서 작업진행해 주시고 마무리되면 push 해주세요 🤘

dipsiiiiiiiiii reviewed (commented):
리뷰 반영 잘 해주셨네요! login javascript 진행해주세요 :)

kangdanne committed:
Add:메인페이지

kangdanne committed:
Add: 레이아웃 수정 및 js구현
- html태그 수정
- css 작업
- 로그인 페이지 js구현
- 메인 페이지 js 구현 중

kangdanne committed:
로그인 & 메인페이지 리팩토링 중

kangdanne committed:
login.js 사용하지 않는 코드 제거

kangdanne committed:
메인 페이지 HTML, CSS 리팩토링 진행중

kangdanne committed:
Add: HTML id, alt속성 수정, css/ js 수정

kangdanne committed:
Modify: 레이아웃 HTML, CSS 수정, JS 구조 수정
- 각 페이지 HTML Head에  viewport 메타태그 추가
- 로그인/ 메인  페이지 JS 기능별로 함수 분리하여 작성
- 로그인 페이지 CSS 비활성화 버튼 커서 스타일 default로 수정
- 메인 페이지 댓글 영역에 삭제 버튼 추가

kangdanne committed:
Modify: main.html 코드 수정, 불필요한 이미지&영상 파일 삭제
- main.html 클로징 안된 <span>태그 닫힘으로 수정
- 사용하지 않는  cats.png, myfeed.mp4파일 제거

kangdanne committed:
Modify: maim.html <img>태그에 중복된 alt값 제거

kangdanne committed:
Modify: 한글 입력시 두 번입력 되는 현상 수정
